apluslms_roman/observer.py

```diff
@@ -151,38 +154,86 @@ def container_msg(self, step, msg):
         self._send_message(Message.CONTAINER_MSG, step, msg)
 
     def result_msg(self, result):
+        state = (StepState.SUCCEEDED if result.ok
+            else (StepState.CANCELLED if result.cancelled
+                else StepState.FAILED))
```

Comment:
Can we have this in format:

```
   state = (StepState.SUCCEEDED if result.ok else
                 StepState.CANCELLED if result.cancelled else
                 StepState.FAILED)
```

I think it would be easier to read.